.github/workflows/ssh-example.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Google LLC
```

Comment:
Do we intend for this to be an example or a test?

Reply:
It's just samples, moved them to examples-workflows directory